docs/UserGuide.md

```diff
@@ -137,6 +141,21 @@ Expected outcome:
 
 <img src="images/list.png" width="550px" height="350px">
 
+### Find a Specific Module : 'find'
+
+Shows the specified module, if it exists in the Current Modules in GradPad.
+
+Module code must be specified and the module must exist in
+the Current Modules in GradPad for successful find.
```

Comment:
Grammar nit: Not really sure how to tweak it but "successful find" sounds a bit weeeird? :grin:
Maybe one way is "...in GradPad to successfully show the found module."

Reply:
Hmmm, what do you think about "...in GradPad for the module to be successfully displayed."? 🤔

Reply:
Yeah okay that sounds good 👍

Reply:
done!